controller.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Dict
+
+from db.iclient_db import IClientDB
+from entities.card import Card
+from entities.card_proprties import CardProperties
+from entities.status import Status
+
+from flask import Flask, request
+from consts import *
+
+
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
+
+
+class Controller:
+    def __init__(self, client_db: IClientDB):
+        self.app = Flask(__name__)
```

Comment:
Use DI, the controller shouldn't know how the app is created

Reply:
BTW what is Flask I heard about it while researching on html to pdf

Reply:
yeah, will be fixed :)
flask is a simple library for http servers in python

Reply:
Didn't you say you wanted to use fast api?